### ISSUE TITLE
Renaming one of the files in android

### ISSUE THREAD

**Issue by Chappie74** [OutdatedGuy/package_rename#1]
Hey. Love the package, does what it is supposed to do. 

However, for android could we also have it rename the .iml file too. 
![image](https://user-images.githubusercontent.com/21108487/186269640-e9117150-a28e-440b-8c8e-09a2bc2646d5.png)

My initial project name is still there, even after running a name change.

I think the naming convention is package_name_android.iml.

**Comment by OutdatedGuy** [comment]
Yo @Chappie74, glad to know you liked the package.

After researching about the `.iml` file I got this proper [answer on stackoverflow](https://stackoverflow.com/a/70509817) which explains that there is no need for any changes to the `.iml` file as it will be auto generated by flutter.

Also the `.iml` file is a project level file and not an app level file. Hence it's name is related to the `name` key in `pubspec.yaml` file rather than the display name of the application itself. So renaming it might break something.

Closing this as not planned.

Feel free to share any additional thoughts on this and I'll reopen the issue.
